graphql/schema.graphql

```diff
@@ -455,16 +529,30 @@ type DocumentPage {
 	paragraphs: [DocumentParagraph!]!
 }
 
+"""
+One paragraph within a [`DocumentPage`]
+"""
 type DocumentParagraph {
 	"""
-	Source text of the paragraph broken down into words
+	Database ID
```

Comment:
Not sure how other people would feel abt this, but at least for me it took a minute to comprehend what this meant. I initially interpreted it at `an id representing a database` instead of the intended `an id for referencing this paragraph within our database`

Reply:
Updated.